<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>650</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string>Read</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="read_tab">
     <attribute name="title">
      <string>Read</string>
     </attribute>
     <widget class="QLabel" name="video_in">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>400</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="data_in">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>771</width>
        <height>201</height>
       </rect>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::WidgetWidth</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QProgressBar" name="read_progress">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>320</y>
        <width>401</width>
        <height>20</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="format">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_start_read">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>161</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Read</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_camera">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="split_size_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Camera:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_camera_update">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>3</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="send_tab">
     <attribute name="title">
      <string>Send</string>
     </attribute>
     <widget class="QPlainTextEdit" name="data_out">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>781</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="video_out">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>140</y>
        <width>450</width>
        <height>450</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_generate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>97</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
     <widget class="QSlider" name="send_slider">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>100</y>
        <width>291</width>
        <height>28</height>
       </rect>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="singleStep">
       <number>10</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickInterval">
       <number>15</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="no_split">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>100</y>
        <width>96</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>No split</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_clear">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="key_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>116</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Key 1</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="key_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>420</y>
        <width>116</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Key 2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="key_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>440</y>
        <width>116</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Key 3</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="key_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>460</y>
        <width>116</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Key 4</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="key_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>480</y>
        <width>116</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Key 5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_save">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="desc_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>116</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Descriptor 1</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="desc_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>116</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Descriptor 3</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="psbt_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>116</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>PSBT 1</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="desc_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>116</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Descriptor 2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="psbt_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>116</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>PSBT 2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="psbt_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>116</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>PSBT 3</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="psbt_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>350</y>
        <width>116</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>PSBT 4</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="psbt_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>116</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>PSBT 5</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_type">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_format">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="steps">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>590</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="split_size">
      <property name="geometry">
       <rect>
        <x>284</x>
        <y>104</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Split size:</string>
      </property>
     </widget>
    </widget>
   </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
